test/unit/factories/anime_weekly_stat.py

```diff
@@ -0,0 +1,29 @@
+import factory
+
+from .session import session_factory
+from .anime import AnimeFactory
+from fal.models import AnimeWeeklyStat
+
+class AnimeWeeklyStatFactory(factory.alchemy.SQLAlchemyModelFactory):
+    class Meta:
+        model = AnimeWeeklyStat
+        sqlalchemy_session = session_factory
+
+    @factory.lazy_attribute
+    def anime_id(self):
+        return self.anime.id
+
+    anime =  factory.SubFactory(AnimeFactory)
+    week = 0
+    watching = factory.Faker('pyint')
+    completed = factory.Faker('pyint')
+    dropped = factory.Faker('pyint')
+    score = factory.Faker('pyfloat')  # min_value/max_value in Faker currently seem broken
+    favorites = factory.Faker('pyint')
+    forum_posts = factory.Faker('pyint')
+    total_points = 0
+
+    class Params:
```

Comment:
Why do we do this instead of assigning `factory.Faker('pyint')` to `total_points`?

Reply:
Originally I left `total_points = 0` and just left it at that, because that's the closest thing to a valid state for anime_weekly_stat (total points isn't a random number its a function of the other stats). When I thought about it some more I thought it could be useful at some point to fake the total points as well, so I left this in as an option for the user.